go/ct/gen/block_context.go

```diff
@@ -175,32 +176,38 @@ func (b *BlockContextGenerator) bindVariablesInRangeConstraints(resultingBlockNu
 	return nil
 }
 
-func (b *BlockContextGenerator) Generate(assignment Assignment, rnd *rand.Rand) (st.BlockContext, error) {
+func (b *BlockContextGenerator) Generate(assignment Assignment, rnd *rand.Rand) (*st.BlockContext, error) {
 	if b.unsatisfiable {
-		return st.BlockContext{}, ErrUnsatisfiable
+		return &st.BlockContext{}, ErrUnsatisfiable
 	}
 
 	// this call takes into account all preassigned values and revision constraints to generate a block number
 	blockNumber, err := b.generateBlockNumber(assignment, rnd)
 	if err != nil {
-		return st.BlockContext{}, err
+		return &st.BlockContext{}, err
 	}
 
 	// for all non bound relevant variables, assign them a value based on the constraints.
 	// 1) fixed offset constraints
 	if err := b.bindVariablesInValueConstraints(assignment, blockNumber); err != nil {
-		return st.BlockContext{}, err
+		return &st.BlockContext{}, err
 	}
 
 	// 2) range constraints
 	if err := b.bindVariablesInRangeConstraints(blockNumber, assignment, rnd); err != nil {
-		return st.BlockContext{}, err
+		return &st.BlockContext{}, err
 	}
 
 	chainId := RandU256(rnd)
 	coinbase, err := RandAddress(rnd)
 	if err != nil {
-		return st.BlockContext{}, err
+		return &st.BlockContext{}, err
+	}
+
+	blobHashes := []vm.Hash{}
+	blobHashesCount := rnd.Intn(st.MaxDataSize)
```

Comment:
That is unreasonably large (MaxDataSize = 1024). Ethereum, right now, has a limit of up to 6 Blobs per transaction. So this limit should be single-digit.